styles/tutanota/catppuccin.user.css

```diff
@@ -13,7 +13,7 @@
 @var select accentSelect "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red*", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
 ==/UserStyle== */
 
-@-moz-document domain("mail.tutanota.com") {
+@-moz-document domain("app.tuta.com") {
```

Comment:
Should include both domains for backwards compatibility

Reply:
That's what I thought but I think it redirects to the new one now. Can you try going to it? I don't have an account with them.

Reply:
👍